app_utils/ipaws_enrichment.py

```
@@ -0,0 +1,230 @@
+        if 'mp3' in mime_type or 'mpeg' in mime_type:
+            ext = '.mp3'
+        elif 'wav' in mime_type:
+            ext = '.wav'
+        elif 'ogg' in mime_type:
+            ext = '.ogg'
+        else:
+            ext = '.mp3'  # IPAWS typically uses MP3
+
+        try:
+            audio_bytes = base64.b64decode(deref_uri)
+        except Exception as exc:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)